### DISCUSSION TITLE
Compile for windows help

### DISCUSSION THREAD

**Discussion by Zyflx** [FunkinCrew/Funkin#1980] [Compiling help]
When I try to compile for windows, I get this C:/HaxeToolkit/haxe/lib/polymod/fs/PolymodFileSystem.hx:19: characters 16-18 : Expected )
Does anyone know how to fix this?

**Comment by ItzOldFlagDEV** [comment]
Check this video: https://www.youtube.com/watch?v=B7tSHaFqyW4 This can fix this.

**Comment by RevixDev** [comment]
I think that might be because the PolyMod may be outdated?
